src/couch_log/src/couch_log_writer_syslog.erl

```diff
@@ -40,6 +40,7 @@
 
 -ifdef(TEST).
 -compile(export_all).
+-compile(nowarn_export_all).
```

Comment:
We don't need `export_all` here. Following functions have explicit export above:
- `init/0`
- `terminate/2`
- `write/2`

Following functions are used only internally (there are no external calls to these functions in the codebase):
- `send/2` used from `write/2`

Functions used from the test suite (see: https://github.com/apache/couchdb/blob/master/src/couch_log/test/couch_log_writer_syslog_test.erl):
- `get_facility/1` 
- `get_level/1` 

Please replace `-compile(export_all)` with:
```
-export([
    get_facility/1, 
    get_level/1
]).

Reply:
Done